connectors/sources/jira.py

```diff
@@ -269,6 +269,7 @@ def get_default_configuration(cls):
             "projects": {
                 "display": "textarea",
                 "label": "Jira project keys",
+                "tooltip": "This configurable field is ignored when Advanced Sync Rules are used.",
```

Comment:
Location here is fine but I notice there's no `order` for `projects`. Could you also add one while you're doing this PR?

Reply:
Updated with commit id: [c44df23](https://github.com/elastic/connectors-python/pull/1025/commits/c44df23c9c71ee41d671df666506142700f46f96)